### ISSUE TITLE
Can't add titles to local

### ISSUE THREAD

**Issue by Liquidacid83** [MALSync/MALSync#1219]
### Describe the Bug

Every time I try to add a new title to local storage now I get a "QUOTA_BYTES quota exceeded" at the bottom but previous ones seem to work just fine.

### Steps to reproduce the behavior

Adding any new title that isn't listed on Mal

### Browser

chrome

### Type

Extension

### URL

mangadex.org

**Comment by lolamtisch** [comment]
Do you have many entries in the local storage? If yes then your extension storage is full as is quite small. Currently there is no solution for that. Our plan is to move the local storage to some generic cloud db provider in the future. But there are still a few things with higher priority so it will still take quite a while until it will be implemented.

**Comment by Liquidacid83** [comment]
Yes since I read a lot of manhwa the like which tend to not be listed on MAL. I'm sure there are a few hundred entries. I'll just have to manually keep track of them from now on then. Thanks for the answer.

**Comment by lolamtisch** [comment]
@Liquidacid83 Another option is to migrate ot anilist, as they have much more manhwa. Or recheck as mal started to add a bunch of them.
